com.logseq.Logseq.yml

```diff
@@ -23,39 +24,39 @@ modules:
   - name: logseq
     buildsystem: simple
     build-options:
-      append-path: /usr/lib/sdk/node16/bin:/usr/lib/sdk/openjdk8/bin
+      append-path: /usr/lib/sdk/node18/bin:/usr/lib/sdk/openjdk17/bin
       env:
         HOME: /run/build/logseq
-        _JAVA_OPTIONS: -Duser.home=/run/build/logseq
+        _JAVA_OPTIONS: "-Duser.home=/run/build/logseq"
         XDG_CACHE_HOME: /run/build/logseq/flatpak-node/cache
         DUGITE_CACHE_DIR: /run/build/logseq/flatpak-node/tmp
         NPM_CONFIG_LOGLEVEL: info
         npm_config_cache: /run/build/logseq/flatpak-node/npm-cache
-        npm_config_offline: 'true'
-        ENABLE_PLUGINS: 'true'
-        ENABLE_FILE_SYNC_PRODUCTION: 'true'
+        npm_config_offline: "true"
+        ENABLE_PLUGINS: "true"
+        ENABLE_FILE_SYNC_PRODUCTION: "true"
     sources:
       - type: file
-        url: https://download.clojure.org/install/linux-install-1.10.1.763.sh
-        sha256: 91421551872d421915c4a598741aefcc6749d3f4aafca9c08f271958e5456e2c
+        url: https://download.clojure.org/install/linux-install-1.11.1.1273.sh
+        sha256: 4938afe6c0690d2a6553141857742d94a7350c02b4fa57cd6a9c1b7cbe66492e
       - type: file
-        url: https://download.clojure.org/install/clojure-tools-1.10.1.763.tar.gz
-        sha256: 2a3ec8a6c3639035c2bba10945ae9007ab0dc9136766b95d2161f354e62a4d10
+        url: https://download.clojure.org/install/clojure-tools-1.11.1.1273.tar.gz
+        sha256: 5f8bafcf24bd148ac9bcbe60a8e7b8093c9ed8eb8e0f387199770e38f0e40ce6
       - type: git
         url: https://github.com/borkdude/rewrite-edn
-        commit: edd87dc7f045f28d7afcbfc44bc0f0a2683dde62
+        commit: f9e71baca3122e04dfc83e14f0ef63da219b0ee6 # v0.4.6
         dest: libs/rewrite-edn
       - type: git
-        url: https://github.com/lread/rewrite-cljc-playground
-        commit: ba1e4255cd1c1fd09dbe2e90556bfa66d5459234
-        dest: libs/rewrite-cljc
+        url: https://github.com/clj-commons/rewrite-clj
+        commit: d0c611e2da6aaddb46ae4f1f0c9531e4b4d3ce7a # v1.1.47
+        dest: libs/rewrite-clj
```

Comment:
Same for this one, it looks like `rewride-edn` does  not use git dependency so we can remove this.